.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,39 @@
+name: Release Instant Seal Node With Deployed Schemas Docker Image
```

Comment:
We want to version with the version of frequency that is being consumed, e.g. if it's 1.1.0 we want v1.1.0 to be reflected in some way of the docker tag for this image